frontend/tests/ui/util.py

```diff
@@ -228,26 +228,28 @@ def create_identities_course(driver, course_alias):
     driver.wait.until(
         EC.element_to_be_clickable(
             (By.XPATH,
-             '//div[@class = "upload-csv"]/div/a'))).click()
-    xpath_table = '//table[starts-with(@class, "identities-table")]/tbody/tr'
-    driver.wait.until(
-        EC.visibility_of_element_located(
-            (By.XPATH, xpath_table)))
-    identity1 = driver.browser.find_element_by_xpath('%s[1]/td/strong' %
-                                                     xpath_table).text
-    password1 = driver.browser.find_element_by_xpath('%s[1]/th' %
-                                                     xpath_table).text
-    identity2 = driver.browser.find_element_by_xpath('%s[2]/td/strong' %
-                                                     xpath_table).text
-    password2 = driver.browser.find_element_by_xpath('%s[2]/th' %
-                                                     xpath_table).text
+             '//div[contains(concat(" ", normalize-space(@class), " "), '
+             '" upload-csv ")]/div/a'))).click()
+
+    usernames_webdriver_elements = driver.browser.find_elements_by_xpath(
+        '//td[contains(concat(" ", normalize-space(@class), " "), " username "'
+        ')]/strong')
+    passwords_webdriver_elements = driver.browser.find_elements_by_xpath(
+        '//td[contains(concat(" ", normalize-space(@class), " "), " password "'
+        ')]')
+    usernames = [username.text for username in usernames_webdriver_elements]
+    passwords = [password.text for password in passwords_webdriver_elements]
+
+    identities = zip(usernames, passwords)
```

Comment:
```suggestion
    identities = [Identity(*x) for x in zip(usernames, passwords)]
```

(donde `Identity` es el namedtuple).

Reply:
Listo